Update README.md (#16)
trane-dev/trane-demos: README.md

<p align="center">
<img width=30% src="https://dai.lids.mit.edu/wp-content/uploads/2018/06/Trane-logo-300x180.jpg" alt=“Trane_Logo” />
</p>

<p align="center">
<i>Trane Demos</i>
</p>

> This repository contains demos that showcase the functionality in the [Trane library](https://github.com/trane-dev/Trane).

Reviewer: ```suggestion
This repository contains demos that showcase the functionality of [Trane](https://github.com/trane-dev/Trane).
```

# Usage

## Google Colab

> Open Google Colaboratory Demo Link from the list below

Reviewer: ```suggestion
Open Google Cola link from the list below:
```

Author: ```suggestion
Open Google Colab link from the list below:
```

```
pip install -r requirements.txt
```
3. Open Jupyter
```
jupyter notebook
```

> 
> 
> ## Citing Trane
> If you use Trane, please consider citing the following paper:
> 
> Ben Schreck, Kalyan Veeramachaneni. [What Would a Data Scientist Ask? Automatically Formulating and Solving Predictive Problems.](https://dai.lids.mit.edu/wp-content/uploads/2017/10/Trane1.pdf) *IEEE DSAA 2016*, 440-451
> 
> BibTeX entry:
> 
> ```bibtex
> @inproceedings{schreck2016would,
>   title={What Would a Data Scientist Ask? Automatically Formulating and Solving Predictive Problems},
>   author={Schreck, Benjamin and Veeramachaneni, Kalyan},
>   booktitle={Data Science and Advanced Analytics (DSAA), 2016 IEEE International Conference on},
>   pages={440--451},
>   year={2016},
>   organization={IEEE}
> }
> ```

Reviewer: Only needed in Trane repo
```suggestion
```
